src/pages/home/report/ReportActionCompose.js

```diff
@@ -288,10 +288,6 @@ class ReportActionCompose extends React.Component {
                                                 onPress={(e) => {
                                                     e.preventDefault();
                                                     this.setMenuVisibility(true);
-
-                                                    /* Keep last focus inside the input so that focus is restored
-                                                     on modal close. Otherwise breaks modal 2 modal transition */
-                                                    this.focus();
```

Comment:
Curious as to what scenarios involve modal-to-modal transitions and whether removing this will break that? 🤔

Reply:
Good point. I believe [this comment](https://github.com/Expensify/Expensify.cash/pull/2114#discussion_r616103088) addresses that concern.

Reply:
Great, thanks for the clarification!